src/flock.rs

```diff
@@ -256,7 +251,7 @@ fn acquire(
     path: &Path,
     lock_try: &dyn Fn() -> io::Result<()>,
     lock_block: &dyn Fn() -> io::Result<()>,
-) -> Result<(), VetError> {
+) -> Result<(), std::io::Error> {
```

Comment:
It feels like this ought to return a `FlockError` if there's any code which returns one here, though perhaps we can get away with just using `io::Error` everywhere in `flock`.